packages/roosterjs-content-model/lib/formatHandlers/block/directionFormatHandler.ts

```diff
@@ -62,6 +67,8 @@ export const directionFormatHandler: FormatHandler<DirectionFormat> = {
 
             if (format.isTextAlignFromAttr) {
                 element.setAttribute('align', value);
+            } else if (element.tagName === 'LI') {
+                element.style.alignSelf = format.textAlign;
```

Comment:
write element.style.display and element.style.flexDirection here as well.

Reply:
The LI element is does not have flexDirection and display, only the UL/OL elements that have